data/tooltips.txt

```diff
@@ -23,6 +23,21 @@ tip "atmosphere scan:"
 tip "bunks:"
 	`How many people (passengers or crew) your ship can hold.`
 
+tip "burn protection:"
+	`Protection against weapons that have a burn effect. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in burn damage, while a total value of 11 only results in a 10 percent reduction.`
+
+tip "burn resistance:"
+	`How much burn damage this ship can resist per second.`
+
+tip "burn resistance energy:"
+	`Energy consumed per second when resisting burn damage.`
+
+tip "burn resistance fuel:"
+	`Fuel consumed per second when resisting burn damage.`
+
+tip "burn resistance heat:"
```

Comment:
Should we add some checks that if the burn damage is less than the generated heat for resisting, that we then don't apply the resistance? (Same for fuel / leak)

Reply:
Remember that burn damage decays with time such that a burn damage of 1 applies 100 heat over time, so if any such check were to be made then we would want to not apply resistance if `burn resistance heat > 100 * burn resistance`, as any amount of heat applied for resisting burn < 100 heat / 1 burn is actually less heat overall. Such a check doesn't really seem necessary to me, though. Whether a burn resistance outfit is actually worth it should be down to implementation.

Reply:
Good point.